src/containers/AccountModal/forms/Registration.tsx

```diff
@@ -57,9 +57,11 @@ const Registration = () => {
         return;
       }
 
-      await register(email, password);
+      const cleanConsentValues = Object.fromEntries(Object.entries(consentValues).filter(isNotEmptyStringEntry).map(formatCrfEntry));
```

Comment:
Should you be using `customerConsents` here? Or do the JWP fields not behave the same way where `publisherConsents` will be a super set of `consentValues`?